### ISSUE TITLE
Add to cart Ipad 3 ios 7 not working

### ISSUE THREAD

**Issue by flody** [wojodesign/simplecart-js#431]
simpleCart.add is not working.

I've tested it on the demo on simplecartjs.org and add to cart does nothing.

Tested with ipad 3rd gen / Ios 7.0.3 / Safari.

It works on chrome.

Any suggestions?

**Comment by tones95** [comment]
I am also having issues with iPad / iOS 7.0 / Mobile Safari.
For some reason you try to add to cart and nothing happens.
I can't personally reproduce this error with my iPad running iOS 7 but I have been on the phone today with countless people who are having this issue.  Terribly frustrating if anyone has a clue as to where to start this would be greatly appreciated.

**Comment by flody** [comment]
Any news about this issue?
@tones95  did you find a solution?

**Comment by tones95** [comment]
Yes, it turned out to be a problem with LocalStorage. If a user had their browser set it Private mode, SimpleCart cannot store the cart.  We added code to fallback to cookies and this resolved the issue.

Tony

**Comment by tzwaeaen** [comment]
@tones95 "We added code to fallback to cookies and this resolved the issue."

I tried it on iOS 8 in private mode, doesn't work. Normal mode works :-(
